<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="0" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-18T00:24:57 IST" name="Default suite" finished-at="2024-10-18T00:25:32 IST" duration-ms="35230">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.runner.RunnerClass@5acf93bb]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2024-10-18T00:24:57 IST" name="Default test" finished-at="2024-10-18T00:25:32 IST" duration-ms="35230">
      <class name="com.runner.RunnerClass">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:com.runner.RunnerClass@5acf93bb]" started-at="2024-10-18T00:24:57 IST" name="setUpClass" finished-at="2024-10-18T00:24:58 IST" duration-ms="1054" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.runner.RunnerClass@5acf93bb]" started-at="2024-10-18T00:24:58 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-10-18T00:25:06 IST" duration-ms="8347" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Add new Customer"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Customers]"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Dashboard / nopCommerce administration] but found [Just a moment...]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Dashboard / nopCommerce administration] but found [Just a moment...]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.stepDef.LoginStep.page_title_should_be(LoginStep.java:41)
at ✽.Page title should be "Dashboard / nopCommerce administration"(file:///C:/Users/letsm/git/ShivaLocalRepo/TestNGWithBDDCucumberFramework-master/src/test/resources/features/Customers.feature:9)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.runner.RunnerClass@5acf93bb]" started-at="2024-10-18T00:25:06 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-10-18T00:25:12 IST" duration-ms="5983" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Successful Login with Valid Credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Login Page]"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Dashboard / nopCommerce administration] but found [Just a moment...]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Dashboard / nopCommerce administration] but found [Just a moment...]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.stepDef.LoginStep.page_title_should_be(LoginStep.java:41)
at ✽.Page title should be "Dashboard / nopCommerce administration"(file:///C:/Users/letsm/git/ShivaLocalRepo/TestNGWithBDDCucumberFramework-master/src/test/resources/features/Login.feature:14)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.runner.RunnerClass@5acf93bb]" started-at="2024-10-18T00:25:12 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-10-18T00:25:19 IST" duration-ms="7101" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Login Data Driven"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Login Page]"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Dashboard / nopCommerce administration] but found [Just a moment...]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Dashboard / nopCommerce administration] but found [Just a moment...]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.stepDef.LoginStep.page_title_should_be(LoginStep.java:41)
at ✽.Page title should be "Dashboard / nopCommerce administration"(file:///C:/Users/letsm/git/ShivaLocalRepo/TestNGWithBDDCucumberFramework-master/src/test/resources/features/Login.feature:28)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.runner.RunnerClass@5acf93bb]" started-at="2024-10-18T00:25:19 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-10-18T00:25:25 IST" duration-ms="6060" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Login Data Driven"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Login Page]"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Dashboard / nopCommerce administration] but found [Just a moment...]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Dashboard / nopCommerce administration] but found [Just a moment...]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.stepDef.LoginStep.page_title_should_be(LoginStep.java:41)
at ✽.Page title should be "Dashboard / nopCommerce administration"(file:///C:/Users/letsm/git/ShivaLocalRepo/TestNGWithBDDCucumberFramework-master/src/test/resources/features/Login.feature:28)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:com.runner.RunnerClass@5acf93bb]" started-at="2024-10-18T00:25:25 IST" name="tearDownClass" finished-at="2024-10-18T00:25:32 IST" duration-ms="6630" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- com.runner.RunnerClass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
